packages/core/src/command/rate-limiting/command/get-tracked-operation-count-sum.ts

```diff
@@ -0,0 +1,52 @@
+import { MetriportError } from "@metriport/shared";
+import { buildDayjs } from "@metriport/shared/common/date";
+import {
+  RateLimitOperation,
+  trackingEntriesSchema,
+} from "@metriport/shared/src/domain/rate-limiting";
+import { DynamoDbUtils } from "../../../external/aws/dynamodb";
+import { Config } from "../../../util/config";
+import { createPrimaryKey } from "../shared";
+
+const region = Config.getAWSRegion();
+
+function validateStartAndEnd(start: string, end: string) {
+  const startJs = buildDayjs(start);
+  if (startJs.millisecond() !== 0)
+    throw new MetriportError("Start is not second granularity", undefined, { start });
+  const endJs = buildDayjs(end);
+  if (endJs.millisecond() !== 0)
+    throw new MetriportError("End is not second granularity", undefined, { start });
+  if (endJs.isBefore(startJs))
+    throw new MetriportError("end is before start", undefined, { start, end });
+}
+
```

Comment:
Is this pretty much functioning as a unit test for the dates? Just cause givent the logic it should always be the case but i suppose this just makes it safer.

Reply:
Yea... it's its own method so want to make sure if it starts getting called elsewhere it validates this. I think there's potentially a shared function that does this essentially, would require me to turn it into a fhir object which isn't really relevant.

Reply:
No longer relevant (using fixed window).